### PR TITLE
Added Simplified Chinese electrum word list

### PR DESCRIPTION
**Pull request by NanoAkron**
List taken from electrum repository at `https://github.com/spesmilo/electrum/tree/master/lib/wordlist`, and trimmed to 1626 words. Original author attribution kept.

List integrity checked by @p1gd0g

Next task: Change language names to native names and (possibly) re-order alphabetically to make it easier to slot-in future additions

### CODE REVIEW

**Comment by moneromooo-monero** [comment]
Thank you. I'd missed that this list was pulled from electrum in the previous PR, and Electrum licence is MIT, so the header file needs to have the licence copied: https://github.com/spesmilo/electrum/blob/master/LICENCE
Note from that licence: The above copyright notice and this permission notice shall be
included in all copies or substantial portions of the Software.
I'd just add the full licence verbatim to be certain all's fine.

**Comment by NanoAkron** [comment]
@moneromooo-monero Will do

**Comment by moneromooo-monero** [comment]
If those actually come from Electrum too, then yes, please do.

**Comment by NanoAkron** [comment]
@moneromooo-monero Japanese definitely does, just checking the others. Please have a look at the  license I've added to the Chinese word list before I force push the others

**Comment by moneromooo-monero** [comment]
Yes, looks good, thank you. I'll let one of the Chinese speakers approve it.

File: src/mnemonics/japanese.h [hunk 1]
```diff
@@ -1,23 +1,43 @@
...
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+// Code surrounding the word list is Copyright (c) 2014-2017, The Monero Project
 // All rights reserved.
-//
```
Comment (NanoAkron) [inline src/mnemonics/japanese.h]: Stupid whitespace changes came with the copy and paste of the amended license wording

**Comment by NanoAkron** [comment]
@p1gd0g would you mind giving this your OK? It's the same list as before.

**Comment by p1gd0g** [comment]
OK.

**Pull request merged**